dockers/docker-platform-monitor/supervisord.conf

```diff
@@ -33,9 +33,11 @@ startsecs=0
 command=/usr/sbin/fancontrol
 priority=4
 autostart=false
-autorestart=false
+autorestart=unexpected
+stopwaitsecs=10
 stdout_logfile=syslog
 stderr_logfile=syslog
+startsecs=10
```

Comment:
10 seconds is the default value for `startsecs` in supervisor, thus this line is unnecessary. However, I am fine with adding the line for the sake of making it explicit, in order to prevent any issues if the supervisor folks change to the default values in the future. However, I would suggest doing it in a separate PR, and adding the line to all relevant files in all applicable places.

Reply:
@jleveque startsecs default value is 1 not 10.
```
startsecs

The total number of seconds which the program needs to stay running after a startup to consider the start successful (moving the process from the STARTING state to the RUNNING state). Set to 0 to indicate that the program needn’t stay running for any particular amount of time.

Note

Even if a process exits with an “expected” exit code (see exitcodes), the start will still be considered a failure if the process exits quicker than startsecs.

Default: 1

Required: No.

Introduced: 3.0
```

Reply:
You are correct, my mistake. I was thinking of a different parameter.